packages/orm/src/EntityManager.ts

```diff
@@ -179,15 +179,14 @@ export class EntityManager<C = unknown> {
   public readonly ctx: C;
   public driver: Driver;
   public currentTxnKnex: Knex | undefined;
-  private _entities: Entity[] = [];
+  #entities: Entity[] = [];
```

Comment:
Moving the mixed "some private _foo", "some private foo", some __data: { foo }" all over to `#foo`.